### PR TITLE
ci: Use ubuntu-latest instead of MacOS for docs checks

### PR DESCRIPTION
**Pull request by barryib**
# PR o'clock

## Description

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by barryib** [comment]
@krrrr38 please review.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -21,18 +21,22 @@ jobs:
...
+      - uses: actions/setup-python@v2
+      - name: Install terraform-docs
         run: |
-          # Temporary fix for https://github.com/actions/virtual-environments/issues/1811
-          brew untap local/homebrew-openssl
-          brew untap local/homebrew-python2
-          brew update
-          brew install pre-commit terraform-docs terraform
+          # Get the download url of the latest version of terraform-docs
+          tf_docs_download_url=$(curl -s https://api.github.com/repos/terraform-docs/terraform-docs/releases/latest | jq -rc '.assets[] | select( .name | contains("linux-amd64")).browser_download_url')
```
Comment (krrrr38) [inline .github/workflows/lint.yml:31]: 📝 terraform-docs recommend way with latest version url

```
> curl -s https://api.github.com/repos/terraform-docs/terraform-docs/releases/latest | jq -rc '.assets[] | select( .name | contains("linux-amd64")).browser_download_url'
https://github.com/terraform-docs/terraform-docs/releases/download/v0.10.1/terraform-docs-v0.10.1-linux-amd64
```

image

**Review by krrrr38 — APPROVED** [review]
lgtm

File: .github/workflows/lint.yml (hunk 1 above)
Comment (barryib) [reply .github/workflows/lint.yml:31]: by using the github api we'll always use the latest stable version of terraform-docs.

**Pull request merged**